doc/manual/types.rst

```diff
@@ -432,7 +434,7 @@ You can also change the values as one would expect:
 .. doctest::
 
     julia> foo.qux = 2
-    2.0
+    2
```

Comment:
This one makes sense, since the value of an assignment is always the original value